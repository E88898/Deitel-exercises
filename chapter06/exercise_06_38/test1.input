501
250
376
439
407
391
383
387
385
384
n

