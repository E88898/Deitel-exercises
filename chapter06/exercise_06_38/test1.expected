I have a number between 1 and 1000.
Can you guess my number?
Please type your first guess. Too high. Try again. Too low. Try again. Too low. Try again. Too high. Try again. Too high. Try again. Too high. Try again. Too low. Try again. Too high. Try again. Too high. Try again. 
Excellent! You guessed the number!
